docs/how-to/debugging.rst

```diff
@@ -268,91 +268,96 @@ HSA provides environment variables that help analyze issues in drivers or hardwa
+        |br| 0x20: Queue commands and queue contents
+        |br| 0x40: Signal creation, allocation, pool
+        |br| 0x80: Locks and thread-safety code
+        |br| 0x100: Copy debug
+        |br| 0x200: Detailed copy debug
+        |br| 0x400: Resource allocation, performance-impacting events
+        |br| 0x800: Initialization and shutdown
+        |br| 0x1000: Misc debug, not yet classified
+        |br| 0x2000: Show raw bytes of AQL packet
+        |br| 0x4000: Show code creation debug
+        |br| 0x8000: More detailed command info, including barrier commands
+        |br| 0x10000: Log message location
+        |br| 0xFFFFFFFF: Log always even mask flag is zero
 
     * - HIP_LAUNCH_BLOCKING
-        | <sub> Used for serialization on kernel execution.</sub>
+        |br|  Used for serialization on kernel execution.
       - 0
       - 0: Disable. Kernel executes normally.
-        | 1: Enable. Serializes kernel enqueue, behaves the same as AMD_SERIALIZE_KERNEL.
+        |br| 1: Enable. Serializes kernel enqueue, behaves the same as AMD_SERIALIZE_KERNEL.
     
     * - HIP_VISIBLE_DEVICES (or CUDA_VISIBLE_DEVICES)
-        | <sub> Only devices whose index is present in the sequence are visible to HIP</sub>
+        |br|  Only devices whose index is present in the sequence are visible to HIP
       -
       - 0,1,2: Depending on the number of devices on the system
 
     * - GPU_DUMP_CODE_OBJECT
-        | <sub>Dump code object</sub>
+        |br| Dump code object
       - 0
       - 0: Disable
-        | 1: Enable
+        |br| 1: Enable
 
     * - AMD_SERIALIZE_KERNEL
-        | <sub> Serialize kernel enqueue</sub>
+        |br|  Serialize kernel enqueue
       - 0
       - 1: Wait for completion before enqueue
-        | 2: Wait for completion after enqueue
-        | 3: Both
+        |br| 2: Wait for completion after enqueue
+        |br| 3: Both
 
     * - AMD_SERIALIZE_COPY
-        | <sub>Serialize copies</sub>
+        |br| Serialize copies
       - 0
       - 1: Wait for completion before enqueue
-        | 2: Wait for completion after enqueue
-        | 3: Both
+        |br| 2: Wait for completion after enqueue
+        |br| 3: Both
 
     * - HIP_HOST_COHERENT
-        | <sub>Coherent memory in hipHostMalloc</sub>
+        |br| Coherent memory in hipHostMalloc
       - 0
       - 0: memory is not coherent between host and GPU
-        | 1: memory is coherent with host
+        |br| 1: memory is coherent with host
 
     * - AMD_DIRECT_DISPATCH
-        | <sub> Enable direct kernel dispatch (Currently for Linux; under development for Windows)</sub>
+        |br| Enable direct kernel dispatch (Currently for Linux; under development for Windows)
       - 1
       - 0: Disable
-        | 1: Enable
+        |br| 1: Enable
 
     * - GPU_MAX_HW_QUEUES
-        | <sub>The maximum number of hardware queues allocated per device</sub>
+        |br| The maximum number of hardware queues allocated per device
       - 4
```

Comment:
Please add one more env variable HIP_LAUNCH_BLOCKING in the table

Reply:
I think it is on line 316 of debugging.rst?